noxfile.py

```diff
@@ -27,9 +27,10 @@
 BLACK_VERSION = "black==19.10b0"
 BLACK_PATHS = ["docs", "google", "tests", "noxfile.py", "setup.py"]
 
-DEFAULT_PYTHON_VERSION = "3.9"
-SYSTEM_TEST_PYTHON_VERSIONS = ["3.9"]
-UNIT_TEST_PYTHON_VERSIONS = ["3.6", "3.7", "3.8", "3.9"]
+DEFAULT_PYTHON_VERSION = "3.8"
+SYSTEM_TEST_PYTHON_VERSIONS = ["3.7"]
+UNIT_TEST_PYTHON_VERSIONS = ["3.6", "3.7", "3.8"]
+
```

Comment:
Quoting @crwilcox from PR #251 

> Look into updating noxfile template.

Reply:
The update should be in synthtool, next gen should have this.